src/components/Accordian/Accordian.tsx

```diff
@@ -0,0 +1,87 @@
+
+import { FC, MouseEvent, SyntheticEvent, useState } from 'react';
+import { classNames } from '../../utils/classNames';
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+//props type
+export interface AccordionProps {
+    /** The title of the accordion */
+    title: string;
+    /** The subtitle of the accordion */
+    subTitle?: string;
+    /** Whether the accordion is disabled or not */
+    disabled?: boolean;
+     /** Whether the accordion is open or not */
+    isOpen?: boolean;
+    /** Callback function triggered when the accordion is toggled */
+    panelId?: string;
+    /** Callback function triggered when the accordion is toggled */
+    children: React.ReactNode;
+    /** Callback function triggered when the accordion is toggled */
+    onPanelChange?: (panel: string) => (event: SyntheticEvent, isExpanded: boolean) => void;
+}
+
+const Accordion: FC<AccordionProps> = ({ 
+  title, 
+  subTitle,
+  disabled, 
+  isOpen: controlledIsOpen, 
+  onPanelChange, 
+  panelId, 
+  children 
+}) => {
+  const [uncontrolledIsOpen, setUncontrolledIsOpen] = useState(false);
+
+  const isOpen = controlledIsOpen ?? uncontrolledIsOpen;
+
+  const handleToggle = (e: MouseEvent) => {
+    e.preventDefault();
+
+    if (disabled) {
+      return;
+    }
+
+    if (onPanelChange) {
+      onPanelChange(panelId)(e, !isOpen);
+    } else {
+      setUncontrolledIsOpen(!isOpen);
+    }
+  };
+
+  let styleClasses = classNames('accordion', {
```

Comment:
In the storybook, you present a group of customized Accordions, but I couldn't find any attributes that you provide to users, that allow them to customize the style.